app/views/handlers/feedback.py

```diff
@@ -174,3 +194,40 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackUpload:
```

Comment:
I'm not sure of the naming here. For me, it's not an upload until we attempt to upload it somewhere. It might we worth considering creating a `FeedbackMetadata` and `FeedbackMessage` class?